src/apps/loyalty/containers/inquiries/index.tsx

```diff
@@ -103,17 +110,30 @@ export class Inquiries extends React.Component<Props, State> {
           <InquiryContainer>
             <Artwork
               artwork={artwork}
-              onSelect={this.onArtworkSelected.bind(this, impulse_conversation_id)}
+              onSelect={this.onArtworkSelected.bind(this, impulse_conversation_id, id)}
             />
           </InquiryContainer>
         </Col>
       )
     })
   }
 
-  onArtworkSelected(conversationId: string, selected: boolean) {
+  onArtworkSelected(conversationId: string, inquiryId: string, selected: boolean) {
+    const selectedConversation = {
+      id: conversationId,
+      inquiry_id: inquiryId,
+    }
+
+    let selectedConversations: SelectedConversation[]
+
+    if (selected) {
+      selectedConversations = this.state.selected_conversations.concat([selectedConversation])
+    } else {
+      selectedConversations = this.state.selected_conversations.filter(val => val.id !== conversationId)
+    }
```

Comment:
Just an FYI: If you’d use the ternary operator here instead, you wouldn’t need to explicitly type `selectedConversations` again.